sdk/benches/sort.rs

```diff
@@ -0,0 +1,72 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion, Throughput};
+use std::time::Duration;
+use surrealdb::dbs::Session;
+use surrealdb::kvs::Datastore;
+use surrealdb_core::sql::Value;
+use tokio::runtime::Runtime;
+
+fn bench_sort(c: &mut Criterion) {
+	let rt = Runtime::new().unwrap();
+
+	let mut group = c.benchmark_group("sort");
+	group.throughput(Throughput::Elements(1));
+	group.sample_size(10);
+	group.measurement_time(Duration::from_secs(15));
+
+	let i = rt.block_on(prepare_data(9999));
+
+	group.bench_function("sort", |b| {
+		b.to_async(Runtime::new().unwrap()).iter(|| run(&i, "SELECT * FROM i ORDER BY v", 9999))
+	});
+
+	group.bench_function("sort-parallel", |b| {
+		b.to_async(Runtime::new().unwrap())
+			.iter(|| run(&i, "SELECT * FROM i ORDER BY v PARALLEL", 9999))
+	});
+
+	let i = rt.block_on(prepare_data(10000));
+
+	group.bench_function("sort-rayon", |b| {
+		b.to_async(Runtime::new().unwrap()).iter(|| run(&i, "SELECT * FROM i ORDER BY v", 10000))
```

Comment:
Just trying to understand why there are 10000 records returned here but 9999 in the above scenario?

Reply:
I am using 9999 and 10000 because, in the code that this PR introduced, we use parallel sorting for resultsets greater than or equal to 10000.
The bench itself doesn't decide which implementation to use, it is the number of records that triggers it.